webapp/src/components/datablocks/MediaBlock.vue

```diff
@@ -8,7 +8,7 @@
       class="mb-3"
       update-block-on-change
     />
-    <img v-if="isPhoto" :src="media_url" class="img-fluid mx-auto" />
+    <img v-if="isPhoto" id="media-block-img" :src="media_url" class="img-fluid mx-auto" />
```

Comment:
```suggestion
    <img v-if="isPhoto" data-testid="media-block-img" :src="media_url" class="img-fluid mx-auto" />
```


Assigning an `id` should be avoided here since `id`s are supposed to be unique across the whole DOM. When two (or more) Media Blocks are added to an Edit page, the images would have the same `id` and that could cause problems. Probably best thing to do here is use a testing-specific attibute, `data-testid="media-block-img"`